feature-group-definitions/webhid.yml

```diff
@@ -0,0 +1,36 @@
+spec: https://wicg.github.io/webhid/
+caniuse: webhid
+usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/2865
+status:
+  is_baseline: false
+  support:
+    chrome: "89"
+    edge: "89"
+compat_features:
+  - api.HID
+  - api.HID.connect_event
+  - api.HID.disconnect_event
+  - api.HID.getDevices
+  - api.HID.requestDevice
+  - api.HIDConnectionEvent
+  - api.HIDConnectionEvent.HIDConnectionEvent
+  - api.HIDConnectionEvent.device
+  - api.HIDDevice
+  - api.HIDDevice.close
+  - api.HIDDevice.collections
+  # - api.HIDDevice.forget
```

Comment:
I'm guessing you omitted this one because it landed several versions later. What do you think of including it anyway, and advancing the Chrome and Edge version numbers? This would make the group more complete to the contemporary API without strongly altering the compatibility story.

Reply:
It's a good question. Should we be more liberal about updating groups for features that are "new"? I'm inclined to say no actually, and think we should handle this situation the same as we would given a new feature that ships in all browsers around the same time, and then gets some small later. I think the feature as a whole should still use the older launch date.

What do about api.HIDDevice.forget then? I don't know. We have this question sprinkled all over our YAML files now.

Reply:
> we should handle this situation the same as we would given a new feature that ships in all browsers around the same time

OK, that's a fair point. I think the overriding thing ought to be developer expectations: if we get information that `forget` is part of the popular understanding of WebHID—even if it arrived later—then it should be part of the main group. But absent that information, it does seem wise to treat it like anything else.

And I'd be probably be less reluctant to split things, if we had already worked out group composition (e.g., "WebHID (kitchen sink)" = "WebHID" + "forget"). So let's keep this as is, then sort out composition so we don't have to have these conversations as often. 😄

Reply:
I agree that developer expectations trumps all.